src/emmy/util/def.cljc

```diff
@@ -220,10 +240,15 @@
   (In ClojureScript, only forms like `(def ~sym ~form)` are emitted, since the
   compiler does not currently error in case 2 and already handles emitting the
   warning for us.)"
-  [#?(:clj ns :cljs _)]
+  [env ns]
   #?(:cljs
```

Comment:
phew, this deserves to be pushed back into potemkin or live somewhere that folks can use it (and invoke grumbling on clojurians slack of course)